routers/web/repo/issue.go

```diff
@@ -896,20 +896,15 @@ func setTemplateIfExists(ctx *context.Context, ctxDataKey string, possibleFiles
 		ctx.Data["label_ids"] = strings.Join(labelIDs, ",")
 		ctx.Data["Reference"] = template.Ref
 		ctx.Data["RefEndName"] = git.RefName(template.Ref).ShortName()
-		return templateErrs
+		return true, templateErrs
 	}
-	return templateErrs
+	return false, templateErrs
 }
```

Comment:
Looking at this function, I have no immediate idea what the bool indicates. I guess this is one case where either a variable for the bool, named return or a comment would be good.